Documentation/architecture/net-platform-standard.md

```diff
@@ -289,136 +290,149 @@ Tooling support for the `netstandard` TFM is as follows. This list will be updat
-| System.Runtime.Serialization.Xml | X | &larr; | &larr; | X | &larr; |
-| System.Runtime.WindowsRuntime | X | &larr; | X | &larr; | &larr; |
-| System.Runtime.WindowsRuntime.UI.Xaml |  | X | &larr; | &larr; | &larr; |
-| System.Security.AccessControl |  |  |  | X | &larr; |
-| System.Security.Claims |  |  |  | X | &larr; |
-| System.Security.Cryptography.Algorithms |  |  |  | X | &larr; |
-| System.Security.Cryptography.Cng |  |  |  | X | &larr; |
-| System.Security.Cryptography.Csp |  |  |  | X | &larr; |
-| System.Security.Cryptography.Encoding |  |  |  | X | &larr; |
-| System.Security.Cryptography.OpenSsl |  |  |  | X | &larr; |
-| System.Security.Cryptography.Primitives |  |  |  | X | &larr; |
-| System.Security.Cryptography.X509Certificates |  |  |  | X | &larr; |
-| System.Security.Principal | X | &larr; | &larr; | &larr; | &larr; |
-| System.Security.Principal.Windows |  |  |  | X | &larr; |
-| System.ServiceModel.Duplex |  | X | &larr; | &larr; | &larr; |
-| System.ServiceModel.Http | X | X | &larr; | X | &larr; |
-| System.ServiceModel.NetTcp |  | X | &larr; | X | &larr; |
-| System.ServiceModel.Primitives | X | X | &larr; | X | &larr; |
-| System.ServiceModel.Security | X | X | &larr; | &larr; | &larr; |
-| System.ServiceProcess.ServiceController |  |  |  | X | &larr; |
-| System.Text.Encoding | X | &larr; | &larr; | X | &larr; |
-| System.Text.Encoding.CodePages |  |  |  | X | &larr; |
-| System.Text.Encoding.Extensions | X | &larr; | &larr; | X | &larr; |
-| System.Text.RegularExpressions | X | &larr; | &larr; | X | &larr; |
-| System.Threading | X | &larr; | &larr; | X | &larr; |
-| System.Threading.AccessControl |  |  |  | X | &larr; |
-| System.Threading.Overlapped |  |  |  | X | &larr; |
-| System.Threading.Tasks | X | &larr; | &larr; | X | &larr; |
-| System.Threading.Tasks.Parallel |  | X | &larr; | &larr; | &larr; |
-| System.Threading.Thread |  |  |  | X | &larr; |
-| System.Threading.ThreadPool |  |  |  | X | &larr; |
-| System.Threading.Timer |  |  | X | &larr; | &larr; |
-| System.Xml.ReaderWriter | X | &larr; | &larr; | X | &larr; |
-| System.Xml.XDocument | X | &larr; | &larr; | X | &larr; |
-| System.Xml.XmlDocument |  |  |  | X | &larr; |
-| System.Xml.XmlSerializer | X | &larr; | &larr; | X | &larr; |
-| System.Xml.XPath |  |  |  | X | &larr; |
-| System.Xml.XPath.XDocument |  |  |  | X | &larr; |
-| System.Xml.XPath.XmlDocument |  |  |  | X | &larr; |
-
+## List of .NET CoreFx APIs and their associated .NET Platform Standard version
+
+### Legend
+- `major.minor.build` - API version `major.minor.build` is supported in this `NETStandard` version.  e.g. In the table below, if you target .NET Platform Standard version 1.4 and reference the System.Runtime package, you'd get the 4.0.20 API version.  If you target .NET Platform Standard version 1.5, you'd get the 4.1.0 API version.
+- `X` - Contract is not supported by this version of `NETStandard`.
```

Comment:
I had an empty cell before and @weshaggard asked me to put an `X` in there.  I'll let folks vote.  @richlander @weshaggard can you cast a vote?